x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotsIntegTests.java

```diff
@@ -451,7 +451,7 @@ public void testMaxRestoreBytesPerSecIsUsed() throws Exception {
             true,
             false,
             IntStream.range(0, nbDocs)
-                .mapToObj(i -> client().prepareIndex(indexName).setSource("foo", randomBoolean() ? "bar" : "baz"))
+                .mapToObj(i -> client().prepareIndex(indexName).setSource("foo", randomAlphaOfLength(1048)))
```

Comment:
does this change make the test more stable?

Reply:
As I tried to explain in initial PR description, I'd like to increase value of `secondsToPause = (bytes / 1024. / 1024.) / mbPerSec` ratio.

Document size (highlighted lines above) eventually translated to `bytes` param in this formula, whereas 90% of minimax goes to `mbPerSec` parameter. By increasing numerator and decreasing denominator we are getting larger value for `secondsToPause`. all these could tigger rate limiting logic more often and hence make test more stable.